webviews/src/App.tsx

```diff
@@ -268,7 +271,7 @@ function App() {
           status={getModelStatus(embeddingsModel)}
           options={embeddingsOptions}
           progress={embeddingsModel ? modelPullProgress[embeddingsModel] : undefined}
-          disabled={!enabled}
+          readOnly={true} // Added readOnly prop here
```

Comment:
I'm not sure this is correct, I need to check

Reply:
yes please. Let me know if any changes are to be made.

Reply:
this makes model selection impossible